adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1843,94 +977,95 @@ void deserialize(final String serializedBlob) throws AuthenticationException {
         }
 
         if (mBrokerProxy.canSwitchToBroker()) {
-            throw new UsageAuthenticationException(ADALError.FAIL_TO_IMPORT,"Failed to import the serialized blob because broker is enabled.");
+            throw new UsageAuthenticationException(ADALError.FAIL_TO_IMPORT, "Failed to import the serialized blob "
+                    + "because broker is enabled.");
         }
 
         final TokenCacheItem tokenCacheItem = SSOStateSerializer.deserialize(serializedBlob);
         final String cacheKey = CacheKey.createCacheKey(tokenCacheItem);
         this.getCache().setItem(cacheKey, tokenCacheItem);  
     }
 
-    class DefaultConnectionService implements IConnectionService {
+    void setIsAuthorityValidated(final boolean isAuthorityValidated) {
+        mIsAuthorityValidated = isAuthorityValidated;
+    }
+
+    boolean getIsAuthorityValidated() {
+        return mIsAuthorityValidated;
+    }
 
-        private Context mConnectionContext;
+    AuthenticationRequestState getWaitingRequest(final int requestId) throws AuthenticationException {
+        Logger.v(TAG, "Get waiting request: " + requestId);
+        AuthenticationRequestState request = null;
 
-        DefaultConnectionService(Context ctx) {
-            mConnectionContext = ctx;
+        mReadLock.lock();
+        try {
+            request = mDelegateMap.get(requestId);
+        } finally {
+            mReadLock.unlock();
         }
```

Comment:
KISS

---
In reply to: [66944722](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/655#discussion_r66944722) [](ancestors = 66944722)